packages/components/src/templates/tab-group.stories.ts

```diff
@@ -0,0 +1,170 @@
+        <sd-tab-panel name="tab-4">
+          <figure class="sd-media pb-4">
+            <img
+              src="./placeholders/images/collaboration.jpg"
+              alt="Two men in business attire sitting on a sofa in a modern office, smiling and looking at a laptop screen."
+              class="aspect-video object-cover"
+            />
+          </figure>
+          <p class="sd-paragraph">
+            Maecenas aliquet nibh nec tempor ultrices. Donec mattis sapien sed elit tincidunt, quis feugiat risus
+            facilisis. Cras rutrum venenatis dui, in sollicitudin nulla vestibulum nec. Maecenas sed imperdiet sapien,
+            non lacinia ex. Etiam rhoncus ullamcorper sem, eu euismod ipsum volutpat non.
+          </p>
+        </sd-tab-panel>
+      </sd-tab-group>`
+};
+
+export const TabWithBadge = {
+  name: 'Tab Group with Badge',
+  render: () =>
+    html` <sd-tab-group activation>
+      <sd-tab slot="nav" panel="tab-1">
+        Notifications
+        <sd-badge class="ml-2">3</sd-badge>
+      </sd-tab>
+      <sd-tab-panel name="tab-1">
+        <h3 class="sd-headline sd-headline--size-3xl">Nisi eu excepteur anim esse</h3>
+        <p class="sd-paragraph mt-2">
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nibh justo ullamcorper odio tempor molestie phasellus
+          dui vel id. Velit in sed non orci pellentesque vivamus nunc. At non tortor, sit neque tristique. Facilisis
+          commodo integer hendrerit tortor.
+        </p>
+      </sd-tab-panel>
+      <sd-tab slot="nav" panel="tab-2">Reminders</sd-tab>
+      <sd-tab-panel name="tab-2">
+        <h3 class="sd-headline sd-headline--size-3xl">Lorem ipsum dolor sit amet</h3>
+        <p class="sd-paragraph mt-2">
+          Maecenas aliquet nibh nec tempor ultrices. Donec mattis sapien sed elit tincidunt, quis feugiat risus
+          facilisis. Cras rutrum venenatis dui, in sollicitudin nulla vestibulum nec. Maecenas sed imperdiet sapien, non
+          lacinia ex. Etiam rhoncus ullamcorper sem, eu euismod ipsum volutpat non.
+        </p>
+      </sd-tab-panel>
+    </sd-tab-group>`
+};
+
+export const TabWithIconBadge = {
+  name: 'Tab Group with Icon and Badge',
+  render: () => html`
+    <sd-tab-group activation>
+      <sd-tab slot="nav" panel="tab-1">
+        <sd-icon slot="left" name="system/dashboard" class="pr-2"></sd-icon>
+        Dashboard
+      </sd-tab>
+      <sd-tab-panel name="tab-1">
+        <p class="sd-paragraph">
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nibh justo ullamcorper odio tempor molestie phasellus
+          dui vel id. Velit in sed non orci pellentesque vivamus nunc. At non tortor, sit neque tristique. Facilisis
+          commodo integer hendrerit tortor.
+        </p>
+      </sd-tab-panel>
+      <sd-tab slot="nav" panel="tab-2">
+        <sd-icon slot="left" name="system/folder"></sd-icon>
+        <sd-badge size="sm" class="mb-3 mr-2" variant="error"></sd-badge>
```

Comment:
Include a status text here, to indicate for screenreaders that the badge exists.

Reply:
Updated